src/readabilityScore/Text.java

```diff
@@ -0,0 +1,115 @@
+package readabilityScore;
+
+public class Text {
```

Comment:
Can we come up with a bit different name reflecting that it's not a text itself, but some statistics calculated base on the text?